cmake/External/oort.cmake

```diff
@@ -0,0 +1,21 @@
+if(NOT __OORT_INCLUDED)
```

Comment:
what exactly is OORT?

Reply:
Oh this is the internal code name of this approach during its experimental phase.
At that time the `mathaot` directory under Triton submodule was a separate repository called "Oort", named after Oort Cloud.

We are planning to release the project as a separate repository and replace the Triton submodule with this new, stand-alone project @malfet
However it takes a while to get approval from AMD to release this project, and we will rename this code name to the final name after the release.